src/hooks/screen/useCreateChat.ts

```diff
@@ -143,9 +139,10 @@ const useCreateChat = () => {
 
       const chatData = createChannelJson(response, selectedUser, true);
       const channelList = ChannelList.fromMessageAnonymouslyAPI(chatData);
-      await channelList.save(localDb);
-      handleMemberSchema(response);
-      goToChatScreen(channelList, GROUP_INFO);
+
+      await Promise.all([channelList.save(localDb), handleMemberSchema(response)]);
+
+      goToChatScreen(channelList as ChannelListObject, GROUP_INFO);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, wrapping the two async operations `channelList.save(localDb)` and `handleMemberSchema(response)` with `Promise.all()` ensures that both operations complete before proceeding. This is a good practice for improving performance by allowing these operations to run concurrently.


<!-- This is an auto-generated comment by OSS CodeRabbit -->